tools/jarcat/main.go

```diff
@@ -219,4 +223,23 @@ func main() {
 	for _, filename := range opts.Zip.In.Get() {
 		must(f.AddFiles(filename))
 	}
+
+	f.Close()
+
+	err = os.Rename(filename, opts.Zip.Out)
+	if err != nil {
+		// Fall back to copy
```

Comment:
can you use `fs.CopyFile` for this? save writing the same code again :)

Reply:
Yup!